2022-03-26 18:32:06,031 INFO    MainThread:29176 [wandb_setup.py:_flush():75] Loading settings from C:\Users\lee\.config\wandb\settings
2022-03-26 18:32:06,031 INFO    MainThread:29176 [wandb_setup.py:_flush():75] Loading settings from E:\AI_project\wandb\settings
2022-03-26 18:32:06,031 INFO    MainThread:29176 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2022-03-26 18:32:06,031 INFO    MainThread:29176 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-03-26 18:32:06,032 INFO    MainThread:29176 [wandb_init.py:_log_setup():405] Logging user logs to E:\AI_project\wandb\run-20220326_183206-1k1sg2yg\logs\debug.log
2022-03-26 18:32:06,032 INFO    MainThread:29176 [wandb_init.py:_log_setup():406] Logging internal logs to E:\AI_project\wandb\run-20220326_183206-1k1sg2yg\logs\debug-internal.log
2022-03-26 18:32:06,032 INFO    MainThread:29176 [wandb_init.py:_jupyter_setup():355] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x0000022533248E80>
2022-03-26 18:32:06,032 INFO    MainThread:29176 [wandb_init.py:init():439] calling init triggers
2022-03-26 18:32:06,032 INFO    MainThread:29176 [wandb_init.py:init():442] wandb.init called with sweep_config: {}
config: {}
2022-03-26 18:32:06,032 INFO    MainThread:29176 [wandb_init.py:init():492] starting backend
2022-03-26 18:32:06,032 INFO    MainThread:29176 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=spawn, using: spawn
2022-03-26 18:32:06,041 INFO    MainThread:29176 [backend.py:ensure_launched():219] starting backend process...
2022-03-26 18:32:06,123 INFO    MainThread:29176 [backend.py:ensure_launched():224] started backend process with pid: 28432
2022-03-26 18:32:06,124 INFO    MainThread:29176 [wandb_init.py:init():501] backend started and connected
2022-03-26 18:32:06,138 INFO    MainThread:29176 [wandb_run.py:_label_probe_notebook():947] probe notebook
2022-03-26 18:32:06,139 INFO    MainThread:29176 [wandb_run.py:_label_probe_notebook():957] Unable to probe notebook: 'gbk' codec can't decode byte 0x80 in position 3308: illegal multibyte sequence
2022-03-26 18:32:06,139 INFO    MainThread:29176 [wandb_init.py:init():565] updated telemetry
2022-03-26 18:32:06,212 INFO    MainThread:29176 [wandb_init.py:init():596] communicating run to backend with 30 second timeout
2022-03-26 18:32:08,485 INFO    MainThread:29176 [wandb_run.py:_on_init():1759] communicating current version
2022-03-26 18:32:08,770 INFO    MainThread:29176 [wandb_run.py:_on_init():1763] got version response 
2022-03-26 18:32:08,771 INFO    MainThread:29176 [wandb_init.py:init():625] starting run threads in backend
2022-03-26 18:32:13,798 INFO    MainThread:29176 [wandb_run.py:_console_start():1733] atexit reg
2022-03-26 18:32:13,798 INFO    MainThread:29176 [wandb_run.py:_redirect():1606] redirect: SettingsConsole.WRAP
2022-03-26 18:32:13,798 INFO    MainThread:29176 [wandb_run.py:_redirect():1643] Wrapping output streams.
2022-03-26 18:32:13,799 INFO    MainThread:29176 [wandb_run.py:_redirect():1667] Redirects installed.
2022-03-26 18:32:13,799 INFO    MainThread:29176 [wandb_init.py:init():664] run started, returning control to user process
2022-03-26 18:32:13,886 INFO    MainThread:29176 [wandb_init.py:_pause_backend():318] pausing backend
2022-03-26 18:32:13,886 INFO    MainThread:29176 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-03-26 19:44:00,760 INFO    MainThread:29176 [wandb_run.py:_atexit_cleanup():1702] got exitcode: 0
2022-03-26 19:44:00,761 INFO    MainThread:29176 [wandb_run.py:_restore():1674] restore
2022-03-26 19:44:01,659 INFO    MainThread:29176 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 825
  total_bytes: 825
}

2022-03-26 19:44:02,066 INFO    MainThread:29176 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 825
  total_bytes: 825
}

2022-03-26 19:44:02,607 INFO    MainThread:29176 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 825
  total_bytes: 26521
}

2022-03-26 19:44:02,717 INFO    MainThread:29176 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 825
  total_bytes: 26521
}

2022-03-26 19:44:02,825 INFO    MainThread:29176 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 825
  total_bytes: 26521
}

2022-03-26 19:44:02,934 INFO    MainThread:29176 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 825
  total_bytes: 26521
}

2022-03-26 19:44:03,041 INFO    MainThread:29176 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 825
  total_bytes: 26521
}

2022-03-26 19:44:03,148 INFO    MainThread:29176 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7636
  total_bytes: 26521
}

2022-03-26 19:44:03,254 INFO    MainThread:29176 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 26521
  total_bytes: 26521
}

2022-03-26 19:44:03,360 INFO    MainThread:29176 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 26521
  total_bytes: 26521
}

2022-03-26 19:44:03,468 INFO    MainThread:29176 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 26521
  total_bytes: 26521
}

2022-03-26 19:44:03,577 INFO    MainThread:29176 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 26521
  total_bytes: 26521
}

2022-03-26 19:44:03,684 INFO    MainThread:29176 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 26521
  total_bytes: 26521
}

2022-03-26 19:44:03,793 INFO    MainThread:29176 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 26521
  total_bytes: 26521
}

2022-03-26 19:44:04,572 INFO    MainThread:29176 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 26521
  total_bytes: 26521
}

2022-03-26 19:44:05,400 INFO    MainThread:29176 [wandb_run.py:_on_finish():1831] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 26521
  total_bytes: 26521
}
local_info {
}

